src/patternfly/components/Switch/switch.scss

```diff
@@ -1,7 +1,7 @@
 .pf-c-switch {
   // Switch
-  --pf-c-switch--Height: calc(#{var(--pf-global--FontSize--md)} * #{var(--pf-global--LineHeight--md)});
-  --pf-c-switch--Width: calc(#{var(--pf-c-switch--Height)} * #{var(--pf-global--golden-ratio)});
+  --pf-c-switch--Height: calc(#{var(--pf-c-switch__label--FontSize)} * #{var(--pf-c-switch--LineHeight)});
+  --pf-c-switch--Width: calc(#{var(--pf-c-switch--Height)} + #{var(--pf-c-switch__label--FontSize)});
```

Comment:
Do you think we should use `--pf-c-switch__toggle--before--Width` instead of `--pf-c-switch__label--FontSize` here? They're the same thing at the moment, but that's really just to create space for the other toggle circle, which would be represented by `--pf-c-switch__toggle--before--Width`

Reply:
yes good idea!